### ISSUE TITLE
External Data Wizard: Tab stops are not intuitive

### ISSUE THREAD

**Issue by vickyharp** [Microsoft/azuredatastudio#3570]
This may be a more general wizard issue, but on keyboard navigation to the Create Database Master Key page, when you enter the page you cannot simply begin typing. You have to back-tab twice to get the focus to the correct location. 

This is one example: keyboard navigation is generally difficult in the wizard right now.

**Comment by vickyharp** [comment]
Dup of https://github.com/Microsoft/azuredatastudio/issues/3079
